amethyst_core/Cargo.toml

```diff
@@ -16,7 +16,9 @@ appveyor = { repository = "amethyst/amethyst" }
 travis-ci = { repository = "amethyst/amethyst" }
 
 [dependencies]
-nalgebra = { version = "0.17", features = ["serde-serialize", "mint"] }
+#nalgebra = { version = "0.17", features = ["serde-serialize", "mint"] }
+nalgebra = { git = "http://github.com/jojolepro/nalgebra", branch = "finite", features = ["serde-serialize", "mint"] }
```

Comment:
We might want to revert this.

Reply:
We don't use N::is_finite(value) anymore?

Reply:
Ah no, probably still used

Reply:
A git dependency will mean we can't release 0.11 until nalgebra releases with the needed feature.

Reply:
nalgebra was release a little while ago

Reply:
Yeah this was fixed in cc955c9. I'm just going to resolve.